b2sdk/account_info/sqlite_account_info.py

```diff
@@ -36,17 +36,39 @@ class SqliteAccountInfo(UrlPoolAccountInfo):
 
     def __init__(self, file_name=None, last_upgrade_to_run=None):
         """
-        If ``file_name`` argument is empty or ``None``, path from ``B2_ACCOUNT_INFO`` environment variable is used. If that is not available, a default of ``~/.b2_account_info`` is used.
+        Initialize SqliteAccountInfo.
+
+        Locations are used in the following order:
```

Comment:
```suggestion
        The exact algorithm used to determine the location of the database file is not API in any sense.
        If the location of the database file is required (for cleanup, etc), do not assume a specific resolution:
        instead, use ``self.filename`` to get the actual resolved location.

        SqliteAccountInfo currently checks locations in the following order:
```
@ppolewicz How does this look?

Reply:
that's way better, thanks